src/databricks/labs/ucx/hive_metastore/tables.scala

```diff
@@ -52,9 +55,19 @@ def metadataForAllTables(databases: Seq[String], queue: ConcurrentLinkedQueue[Ta
                 s"$key=$value"
           }.mkString("[", ", ", "]")
 
+          // Note: sharedState.externalCatalog does not expose accurate partition metadata, so it forced us to use sessionState.catalog instead
+          val partitionColumnNames = try {
+            spark.sessionState.catalog.getTableMetadata(TableIdentifier(tableName, Some(databaseName))).partitionColumnNames
+          }
+          catch {
+            case e: Exception => null
+          }
+          val isPartitioned = if (partitionColumnNames != null && !partitionColumnNames.isEmpty) true else false
```

Comment:
Can you add an integration test in test_installation.py, where you create a partitioned table via sql_backend, run assessment workflow, and check if the correct result comes back in tables_crawler.snapshot()